src/rewrite_clj/node/namespaced_map.clj

```diff
@@ -0,0 +1,76 @@
+(ns ^:no-doc rewrite-clj.node.namespaced-map
+  (:require
+   [clojure.string :as str]
+   [rewrite-clj.node.protocols :as node]))
+
+;; ## Node
+
+(defn- assert-namespaced-map-children
+  [children]
+  (let [exs (node/sexprs children)]
+    (assert (= (count exs) 2)
+            "must contain 2 non-whitespace forms.")
+    (assert (keyword? (first exs))
+            "first form in namespaced must be a keyword.")
```

Comment:
Shouldn't this be "... in namespaced *map* must ..." ?

Reply:
Yes indeed! Thanks so much for reviewing, @pedro-w!